frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -268,41 +267,17 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
         }
 
         if (decision instanceof Decision.TemporarilyLockedOut temporarilyLockedOut) {
-            if (isReauthJourney) {
-                auditService.submitAuditEvent(
-                        FrontendAuditableEvent.AUTH_REAUTH_FAILED,
-                        auditContext.withSubjectId(authSession.getInternalCommonSubjectId()),
-                        ReauthMetadataBuilder.builder(calculatedPairwiseId)
-                                .withAllIncorrectAttemptCounts(
-                                        getReauthAttemptCounts(
-                                                journeyType,
-                                                userProfile.getSubjectID(),
-                                                calculatedPairwiseId))
-                                .withFailureReason(ReauthFailureReasons.INCORRECT_PASSWORD)
-                                .build());
-                cloudwatchMetricsService.incrementCounter(
-                        CloudwatchMetrics.REAUTH_FAILED.getValue(),
-                        Map.of(
-                                ENVIRONMENT.getValue(),
-                                configurationService.getEnvironment(),
-                                FAILURE_REASON.getValue(),
-                                ReauthFailureReasons.INCORRECT_PASSWORD.getValue()));
-
-                return generateApiGatewayProxyErrorResponse(
-                        400, ErrorResponse.TOO_MANY_INVALID_REAUTH_ATTEMPTS);
-            } else {
-                auditService.submitAuditEvent(
-                        FrontendAuditableEvent.AUTH_ACCOUNT_TEMPORARILY_LOCKED,
-                        auditContext,
-                        pair(INTERNAL_SUBJECT_ID, userProfile.getSubjectID()),
-                        pair(
-                                AuditableEvent.AUDIT_EVENT_EXTENSIONS_ATTEMPT_NO_FAILED_AT,
-                                configurationService.getMaxPasswordRetries()),
-                        pair(NUMBER_OF_ATTEMPTS_USER_ALLOWED_TO_LOGIN, incorrectPasswordCount));
-
-                return generateApiGatewayProxyErrorResponse(
-                        400, ErrorResponse.TOO_MANY_INVALID_PW_ENTERED);
-            }
+            auditService.submitAuditEvent(
```

Comment:
This change also mean the `authenticationAttemptsService` can also be removed from this handler 🎉